firestore/firestore_snippets/collection_group_query.go

```diff
@@ -51,3 +51,66 @@ func collectionGroupQuery(w io.Writer, projectID string) error {
 }
 
 // [END firestore_query_collection_group_filter_eq]
+
+func partitionQuery(ctx context.Context, client *firestore.Client) error {
+	// [START firestore_partition_query]
+	cities := client.CollectionGroup("cities")
+
+	// Get a partioned query for the cities collection group, with a maximum
+	// partition count of 10
+	partitionedQueries, err := cities.GetPartitionedQueries(ctx, 10)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
do we want to just print here? Should we throw instead?

Reply:
I think I borrowed this from elsewhere. As the sample region doesn't include the func, it is odd we are returning from, essentially, nowhere. Though I saw other places that write to writer (use `Fprintln`) which could have testing advantages. Though I just noticed on ln 77 I do return err making this even odder. @tbpg I'd love your thoughts here if you want to weigh in. Then I can make this all consistent.

Reply:
Perfect. Thank you for your input here. You are correct, those samples are all pretty light. That may be something to address, but it seems a more holistic thing than starting with this one sample.